scripts/upload.ts

```diff
@@ -10,123 +10,126 @@ args:
   - --import-map={{ srcroot }}/import-map.json
 ---*/
 
-import { S3 } from "s3"
-import { pkg as pkgutils } from "utils"
-import { useFlags, useOffLicense, useCache, usePrefix } from "hooks"
-import { Package, PackageRequirement } from "types"
-import SemVer, * as semver from "semver"
-import { dirname, basename } from "deno/path/mod.ts"
-import Path from "path"
-import { set_output } from "./utils/gha.ts"
-import { sha256 } from "./bottle.ts"
-
-useFlags()
-
-if (Deno.args.length === 0) throw new Error("no args supplied")
-
-const s3 = new S3({
-  accessKeyID: Deno.env.get("AWS_ACCESS_KEY_ID")!,
-  secretKey: Deno.env.get("AWS_SECRET_ACCESS_KEY")!,
-  region: "us-east-1",
-})
-
-const bucket = s3.getBucket(Deno.env.get("AWS_S3_BUCKET")!)
-const encode = (() => { const e = new TextEncoder(); return e.encode.bind(e) })()
-const cache = useCache()
-
-const pkgs = args_get("pkgs").map(pkgutils.parse).map(assert_pkg)
-const srcs = args_get("srcs")
-const bottles = args_get("bottles")
-const checksums = args_get("checksums")
-
-
+import { S3, S3Bucket } from "s3";
+import { pkg as pkgutils } from "utils";
+import { useCache, useFlags, useOffLicense, usePrefix } from "hooks";
+import { Package, PackageRequirement } from "types";
+import SemVer, * as semver from "semver";
+import { basename, dirname } from "deno/path/mod.ts";
+import Path from "path";
+import { set_output } from "./utils/gha.ts";
+import { sha256 } from "./bottle.ts";
```

Comment:
Our current code style doesn't end lines with semi-colons (which accounts for the excess changes). Recommend removal to minimize changeset.

Reply:
Sorry @jhheider, my fault: I didn't consider that. I removed all semicolons